nucliadb_node/src/reader/grpc_driver.rs

```diff
@@ -35,7 +36,14 @@ impl From<NodeReaderService> for NodeReaderGRPCDriver {
         NodeReaderGRPCDriver(RwLock::new(node))
     }
 }
-
+impl NodeReaderGRPCDriver {
+    async fn shard_loading(&self, id: &ShardId) {
```

Comment:
Can you document this method please ?

Reply:
Sure, done at [cf87f88](https://github.com/nuclia/nucliadb/pull/375/commits/cf87f88b2ec66bbd265faaff8a884e8702b9d80d).